indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -162,7 +165,25 @@ def _process(self, action, comment):
             argmap['files'] = EditingFilesField(self.event, self.contrib, self.editable_type, allow_claimed_files=True,
                                                 required=True)
         args = parser.parse(argmap)
-        review_editable_revision(self.revision, session.user, action, comment, args['tags'], args.get('files'))
+        service_url = editing_settings.get(self.event, 'service_url')
+
+        revision = review_editable_revision(self.revision, session.user, action, comment, args['tags'],
+                                            args.get('files'))
+        publish = True
+        db.session.commit()
+        if service_url:
+            try:
+                resp = service_handle_review_editable(self.editable, revision or self.revision)
+                publish = resp.get('publish', True)
+            except ServiceRequestFailed:
+                self.revision.final_state = FinalRevisionState.none
+                if revision:
+                    delete_revision(revision)
+                db.session.commit()
+                raise ServiceUnavailable(_('Revision failed, please try again later.'))
```

Comment:
I guess since we raise an exception here, the email queued for sending in `notify_editor_judgment` won't be sent?

Reply:
Great question. I had a note here to do the `notify_editor_judgment` within the RH because I thought notifications were probably being sent. But even better now that I found out about the queue, looks like raising an exception here doesn't even let it get to the `flush_emails` part!